src/main/java/dev/vasyl/proj/service/BookServiceImpl.java

```diff
@@ -34,4 +34,14 @@ public BookDto findById(long id) {
                 () -> new EntityNotFoundException("Can`t find Book entity by id: " + id));
         return bookMapper.toDto(book);
     }
+
+    @Override
+    public BookDto update(Long id, CreateBookRequestDto requestDto) {
+        bookRepository.findById(id).orElseThrow(
+                () -> new EntityNotFoundException("Can't find Book entity by id: " + id));
+        Book book = bookMapper.toModel(requestDto);
+        book.setId(id);
```

Comment:
Create method in mapper
```suggestion
        bookMapper.updateBookFromDto(requestDto, book);
```